charts/telemetry-collector/values.yaml

```diff
@@ -0,0 +1,167 @@
+            "MMHoPrepInterFail": "handover_and_mobility_metrics",
+            "MMHoResAlloInterReq": "handover_and_mobility_metrics",
+            "MMHoResAlloInterSucc": "handover_and_mobility_metrics",
+            "MMHoResAlloInterFail": "handover_and_mobility_metrics",
+            "MMHoExeInterReq": "handover_and_mobility_metrics",
+            "AccDRBEstabAtt5QiSnssai": "radio_resource_utilization_metrics",
+            "DRBEstabSuccSnssai": "radio_resource_utilization_metrics",
+            "DRBEstabSucc5Qi": "radio_resource_utilization_metrics",
+            "DRBRelActNbr5QiSnssai": "radio_resource_utilization_metrics",
+            "DRBRelActNbrSnssai": "radio_resource_utilization_metrics",
+            "DRBRelActNbr5Qi": "radio_resource_utilization_metrics",
+            "MMHoExeInterReqTimeMean": "handover_and_mobility_metrics",
+            "MMHoExeInterReqTimeMax": "handover_and_mobility_metrics",
+            "AccGTPThpDl": "throughput_metrics",
+            "AccGTPThpDlQfiSnssaiPlmn": "throughput_metrics",
+            "AccGTPThpUl": "throughput_metrics",
+            "AccGTPThpUlQfiSnssaiPlmn": "throughput_metrics",
+            "AccDRBPdcpSduDelayDlQfiSnssaiPlmn": "radio_resource_utilization_metrics",
+            "AccDRBF1UPacketLossRateUlQoS": "radio_resource_utilization_metrics",
+            "AccDRBF1UPacketLossRateUlQfiSnssaiPlmn": "radio_resource_utilization_metrics",
+            "AccDRBF1UPacketLossRateUlSnssai": "radio_resource_utilization_metrics",
+            "AccDRBPdcpPacketDropRateDlQoS": "radio_resource_utilization_metrics",
+            "AccDRBPdcpPacketDropRateDlSnssai": "radio_resource_utilization_metrics",
+            "AccDRBPdcpPacketDropRateDlQfiSnssaiPlmn": "radio_resource_utilization_metrics",
+            "DRBPdcpSduDelayDlQoS": "radio_resource_utilization_metrics",
+            "DRBPdcpSduDelayDlSnssai": "radio_resource_utilization_metrics",
+            "DRBPdcpSduDelayDlDistQos": "radio_resource_utilization_metrics",
+            "DRBPdcpSduDelayDlDist": "radio_resource_utilization_metrics",
+            "DRBPdcpSduDelayDlDistSnssai": "radio_resource_utilization_metrics",
+            "AccDRBPdcpSduDelayDlDistQfiSnssaiPlmn": "radio_resource_utilization_metrics"
+          }
+  
+    - name: "acc_du"
+      component: "du"
+      vendor: "accelleran"
+      enabled: "true"
+      consumers:
+        udpConsumer:
+          port: "55555"
+      publishers:
+        kafkaPublisher:
+          topic: "accelleran.drax.5g.ric.o1.ves"
+        influxPublisher:
+          defaultBucket: "acc_du"
+          buckets: {
+            "DUUEMeasurements": "du_ue_measurements",
+            "DURLCMeasurements": "du_rlc_measurements"
+          }
+
+      
+  influxDB:
+    url: "http://du-influxdb2:8086"
+    token: "yMq5frlbmU_0CZYqPphL0hDY35V_Z2PRpvkdySyfbl5-B70gs2zN03rJt0hPBi565HsRRF4J-RJ_nzLntNPTRA=="
+    org: "accelleran"
+
+influxDBToken: "UcmOqrU5834fniAi6cRfCLa3OkxZrh9Y"
```

Comment:
Don't expose actual tokens

Reply:
Yes - spotted this and I had deleted them!